proc_macros/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "proc_macros"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+Inflector = "0.11.4"
```

Comment:
A capitalized crate name?

Almost seems ironic since what this crate is doing is case conversion

Reply:
Ha, maybe it's a weird joke.